Automattic/woocommerce-payments
# Wip: Fix - deposit and transaction count for table summary

Fixes #1639 

#### Changes proposed in this Pull Request

Presently we render the number of transactions and deposits in the summary section of the respective tables. While the data is being fetched from the server the count is rendered as `undefined`.

I have set the default value of count as zero, in case the count in not available (while loading or when API call fails)

#### Testing instructions
- Checkout to `develop` branch.
- Try loading the depostis and transactions table, you'll see the count as `undefined` at the bottom of the table (table summary) 
- now checkout to the branch `fix/transaction-deposit-summary`
- try doing the same, you'll see the count as zero while the loading state.

#### Before
<img width="973" alt="Screenshot 2021-04-21 at 3 42 00 AM" src="https://user-images.githubusercontent.com/15019298/115471350-7263e000-a255-11eb-977d-4332c19c7ca1.png">

#### After

<img width="987" alt="Screenshot 2021-04-22 at 4 21 12 AM" src="https://user-images.githubusercontent.com/15019298/115630862-6560f280-a322-11eb-837e-0d49b691380f.png">
